activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/IdentityRuntimeService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.identity;
+
+import java.util.List;
+import org.activiti.cloud.identity.model.User;
+
+public interface IdentityRuntimeService {
```

Comment:
Could you rename to IdentityService, please can leave a comment on IdentityManagerService saying that that interface should be used only in Identity Adapter Service and not for other purposes, while for genreral purpose we should use this class.